modules/cassandra/metadata.yaml

```diff
@@ -168,142 +212,192 @@ metrics:
+          chart_type: line
           dimensions:
             - name: used
         - name: cassandra.compaction_completed_tasks_rate
+          availability: []
           description: Completed compactions rate
           unit: tasks/s
+          chart_type: line
           dimensions:
             - name: completed
         - name: cassandra.compaction_pending_tasks_count
+          availability: []
           description: Pending compactions
           unit: tasks
+          chart_type: line
           dimensions:
             - name: pending
         - name: cassandra.compaction_compacted_rate
+          availability: []
           description: Compaction data rate
           unit: bytes/s
+          chart_type: line
           dimensions:
             - name: compacted
         - name: cassandra.jvm_memory_used
+          availability: []
           description: Memory used
           unit: bytes
+          chart_type: stacked
           dimensions:
             - name: heap
             - name: nonheap
         - name: cassandra.jvm_gc_rate
+          availability: []
           description: Garbage collections rate
           unit: gc/s
+          chart_type: line
           dimensions:
             - name: parnew
             - name: cms
         - name: cassandra.jvm_gc_time
+          availability: []
           description: Garbage collection time
           unit: seconds
+          chart_type: line
           dimensions:
             - name: parnew
             - name: cms
         - name: cassandra.dropped_messages_rate
+          availability: []
           description: Dropped messages rate
           unit: messages/s
+          chart_type: line
           dimensions:
             - name: dropped
         - name: cassandra.client_requests_timeouts_rate
+          availability: []
           description: Client requests timeouts rate
           unit: timeout/s
+          chart_type: line
           dimensions:
             - name: read
             - name: write
         - name: cassandra.client_requests_unavailables_rate
+          availability: []
           description: Client requests unavailable exceptions rate
           unit: exceptions/s
+          chart_type: line
           dimensions:
             - name: read
             - name: write
         - name: cassandra.client_requests_failures_rate
+          availability: []
           description: Client requests failures rate
           unit: failures/s
+          chart_type: line
           dimensions:
             - name: read
             - name: write
         - name: cassandra.storage_exceptions_rate
+          availability: []
           description: Storage exceptions rate
           unit: exceptions/s
+          chart_type: line
           dimensions:
             - name: storage
     - name: thread pool
-      description: Metrics related to Cassandra's thread pools. Each thread pool provides its own set of the following metrics.
+      description: ''
```

Comment:
?

Reply:
well, just forgot that, will add it back in